layouts/partials/head.html

```diff
@@ -114,17 +114,17 @@
     {{ end }}
 
     {{/* Tests to enforce required frontmatter for certain content types. */}}
-    {{ if eq .Type "blog" }}
-        {{ if not (isset .Page.Params "authors") }}
+    {{ if and (eq .Type "blog") (.IsPage) }}
```

Comment:
Nit: Do we need parens around `.IsPage`?

Reply:
No, it just looked cleaner to my eyes with the parens to the left. But since it's not necessary and you pointed it out, I've removed.